eessi_bot_job_manager.py

```diff
@@ -667,14 +674,26 @@ def main():
     if max_iter != 0:
         known_jobs = job_manager.get_known_jobs()
     while max_iter < 0 or i < max_iter:
+        # sleep poll_interval seconds (not for the first iteration)
+        if i != 0 and (max_iter < 0 or i < max_iter):
```

Comment:
Could we simplify the logical expression to the following?
```suggestion
        if i != 0:
```
Should be ok because the loop body is only entered if `max_iter < 0 or i < max_iter` evaluates to `True`.